bundles/admin/donationProcessing/readDonations.tsx

```diff
@@ -31,27 +31,61 @@ export default React.memo(function ReadDonations() {
   const { ADMIN_ROOT } = useConstants();
   const { event: eventId } = useParams<{ event: string }>();
   const status = useSelector((state: any) => state.status);
+  const bids = useSelector((state: any) => state.models.bid);
+  const donationbids = useSelector((state: any) => state.models.donationbid);
   const donations = useSelector((state: any) => state.models.donation);
   const donors = useSelector((state: any) => state.models.donor);
   const event = useSelector((state: any) => state.models.event?.find((e: any) => e.pk === +eventId!));
   const dispatch = useDispatch();
   const canEditDonors = usePermission('tracker.change_donor');
+  const fetchBids = useCallback(
+    (e?: React.MouseEvent<HTMLButtonElement>) => {
+      const bidparams = {
+        event: eventId,
+      };
+
+      dispatch(modelActions.loadModels('bid', bidparams));
+
+      e?.preventDefault();
+    },
+    [dispatch, eventId],
+  );
+  const fetchDonationBids = useCallback(
+    (e?: React.MouseEvent<HTMLButtonElement>) => {
+      const donobidparams = {
+        event: eventId,
+      };
+
+      dispatch(modelActions.loadModels('donationbid', donobidparams));
+
+      e?.preventDefault();
+    },
+    [dispatch, eventId],
+  );
```

Comment:
Questo schifo mostruoso è l'unico modo in cui sono riuscito a fetchare i dati dipendenti in maniera sincronizzata ma apparentemente non c'è verso di stabilire una catena di dipendenze (e.g. voglio caricare prima bids, poi donations, poi donation bids perché ciascuno dipende da quello prima) per cui ogni tanto ci scappano degli `a is undefined` a caso se al caricamento della pagina le dono finiscono prima delle bids ecc. Idee su come gestire sta roba? Ai miei tempi si era inventato async/await ma vedo che apparentemente non va più di moda. Penso sia un problema di come si sono inventati il caricamento dei modelli, veramente raccapricciante.

cc @f4nu